### ISSUE TITLE
GA process output

### ISSUE THREAD

**Issue by mahmodDAHOL** [ahmedfgad/GeneticAlgorithmPython#154]
Is there a parameter or whatever it is for showing output for every step in process of GA?

**Comment by ahmedfgad** [comment]
You can use the callback functions to print something about the progress. Please check this example: https://pygad.readthedocs.io/en/latest/README_pygad_ReadTheDocs.html#life-cycle-of-pygad

**Comment by whubaichuan** [comment]
@ahmedfgad I think the workflow may be wrong. I have tested and show my point of view.
<img width="920" alt="1" src="https://user-images.githubusercontent.com/28725662/220398099-1709f415-4f96-4a49-9666-7906095d5b52.png">

**Comment by ahmedfgad** [comment]
@whubaichuan,

Yes you are right. The library went through some changes which affected some steps. This is the new lifecycle.

![PyGAD Lifecycle](https://user-images.githubusercontent.com/16560492/220486073-c5b6089d-81e4-44d9-a53c-385f479a7273.jpg)
